tests/AdminTest.php

```diff
@@ -45,7 +50,7 @@ public function anon_user_gets_login_form_on_admin(): void
     /**
      * @test
      */
-    public function feed_index_loads(): void
+    public function feedIndexLoads(): void
```

Comment:
Lately I've found the snake-case style for test methods to be easier to read.  I've seen lots of recommendations in every direction on this; I'd rather not mess with this along with everything else.